suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml

```diff
@@ -0,0 +1,3 @@
+tasks:
```

Comment:
what I am saying is 2-setup folder is not required if you add this task to the workloads itself, also this I think can get read of the folder prefix 0,1,2,3 etc